### ISSUE TITLE
Allow for delivery via generic mail servers

### ISSUE THREAD

**Issue by danielquinn** [tomav/docker-mailserver#421]
I have been looking for this sort of project for a long time, so I'm just thrilled to have stumbled upon it this morning.  Unfortunately, the one use case I need it for is not yet supported: I want to have this mailserver run on a local network, accept mail for delivery and then *have an external mail service do the actual delivery*.  I see the wiki has this page about [doing this with AWS](https://github.com/tomav/docker-mailserver/wiki/Configure-AWS-SES) but it seems very AWS-centric and the mail host I have in mind is [Webfaction](https://webfaction.com/).

I'm going to try to make this work regardless, but it would be nice if there was a more generic way to do this with environment variables.  Something like `RELAYHOST=`, `RELAYUSER=`, and `RELAYPASS=` or something.

**Comment by tomav** [comment]
Try AWS config, it may work when I look at the code.
Let us know.

**Comment by tomav** [comment]
Hi @danielquinn, any update on this? Thank, you.

**Comment by danielquinn** [comment]
I gave it a shot, but I'm afraid I don't know enough about mail servers and the various technologies you've employed here to make use of it.  At one point, I managed to get the server configured & started, and it did relay mail to Webfaction, but messages to my other Webfaction-hosted accounts never appeared, and messages to my Gmail address arrived tagged with a `?` and a message that Google couldn't confirm that this message was from a legit server, but also couldn't be sure it was spam.

I tried restarting everything and now Docker is complaining that it can't read a file:

    /tmp/docker-mailserver/opendkim/keys/<my domain>/mail.private

which makes sense because it's owned by root.  It would have been created by the DKIM portion of the setup -- something I honestly know nothing about and don't know if I can do without.

So I've done what I can, but I think this is over my head.  Sorry.
